frontend/src/modules/entitydetails/components/ContextIssueButton.js

```diff
@@ -0,0 +1,36 @@
+/* @flow */
+
+import * as React from 'react';
+import { Localized } from '@fluent/react';
+
+import './ContextIssueButton.css';
+
+type Props = {|
+    openTeamComments: () => void,
+|};
+
+export default function ContextIssueButton(props: Props) {
+    const [isDisabled, setIsDisabled] = React.useState(false);
+
+    const handleClick = () => {
+        setIsDisabled(true);
+        props.openTeamComments();
+
+        setTimeout(() => {
+            setIsDisabled(false);
+        }, 3000);
```

Comment:
I don't think that's a reliable solution. What if I click after 3 seconds? Also, if I move to the next string quickly, the button remains blocked. We should instead check if the mention already exists and not insert it again.